src/api/services/emailService.ts

```diff
@@ -0,0 +1,17 @@
+import { EmailArgs } from './emailTypes';
+import * as NodemailerService from './nodemailerService';
+import * as SendGridService from './sendGridService';
+
+export function createEmailService() {
+  const isProduction = process.env.NODE_ENV === 'production';
```

Comment:
will integration environment still use sendgrid? (as node env is production in integ too)

Reply:
Yes, I think integration should use a different SendGrid account to test the email workflow